src/components/MessagesList/MessagesGroup/Message/MessagePart/MessageBody.vue

```diff
@@ -426,12 +459,51 @@ export default {
 		containsCodeBlocks() {
 			return this.message.message.includes('```')
 		},
+
+		iconMessageDeliverySize() {
+			return this.isSplitViewEnabled ? 14 : 16
+		},
+
+		isOwnMessage() {
+			return this.actorStore.checkIfSelfIsActor(this.message) && !this.isSystemMessage
+		},
+
+		isShortSimpleMessage() {
```

Comment:
There is nothing in this computed, that isn't know (or can't be computed) in MessageItem.vue